lms/djangoapps/onboarding_survey/forms.py

```diff
@@ -428,10 +428,10 @@ class Meta:
         }
 
     def clean_organization_name(self):
-        is_currently_unemployed = True if self.data.get('is_currently_employed') == 'true' else False
+        is_currently_employed = True if self.data.get('is_currently_employed') == 'true' else False
```

Comment:
we can write this like this
`is_currently_employed = self.data.get('is_currently_employed') == 'true'`